app/src/main/java/com/gh4a/utils/HttpImageGetter.java

```diff
@@ -457,36 +452,23 @@ private Drawable loadImageForUrl(String source) {
                         mime = URLConnection.guessContentTypeFromStream(is);
                     }
                     if (mime != null && mime.startsWith("image/svg")) {
-                        bitmap = renderSvgToBitmap(mContext.getResources(), is, mWidth, mHeight);
+                        bitmap = renderSvgToBitmap(mContext.getResources(), is);
                     } else {
                         boolean isGif = mime != null && mime.startsWith("image/gif");
                         if (!isGif || canLoadGif()) {
-                            output = File.createTempFile("image", ".tmp", mCacheDir);
-                            if (FileUtils.save(output, is)) {
-                                if (isGif) {
-                                    GifDrawable d = new GifDrawable(output);
-                                    d.setBounds(0, 0, d.getIntrinsicWidth(), d.getIntrinsicHeight());
-                                    return d;
-                                } else {
-                                    bitmap = getBitmap(output, mWidth, mHeight);
-                                }
+                            if (isGif) {
```

Comment:
Can probably save one indention level now:
```
if (isGif && canLoadGif()) {
    ...
} else if (!isGif) {
    ...
}

Reply:
Done. The condition is now much more easily understandable IMO